src/modules/random/index.ts

```diff
@@ -230,7 +230,7 @@ export class RandomModule {
   /**
    * Generating a string consisting of letters in the English alphabet.
    *
-   * @param options Either the number of characters or an options instance. Defaults to `{ count: 1, casing: 'mixed', bannedChars: [] }`.
+   * @param options Either the number of characters or an options instance.
```

Comment:
Removed for improved docs readability. The value will be ignored by the docs generation anyway.

Reply:
This should really be:

```suggestion
   * @param options Either the number of characters or an options instance. Defaults to `{}`.
```

To be consistent with other places in our codebase. Examples:
- https://github.com/faker-js/faker/blob/ee12c32fced6cdf6fef2c5993af84825fd88606f/src/modules/airline/index.ts#L141
- https://github.com/faker-js/faker/blob/ee12c32fced6cdf6fef2c5993af84825fd88606f/src/modules/commerce/index.ts#L58
- https://github.com/faker-js/faker/blob/ee12c32fced6cdf6fef2c5993af84825fd88606f/src/modules/finance/index.ts#L410
- https://github.com/faker-js/faker/blob/ee12c32fced6cdf6fef2c5993af84825fd88606f/src/modules/internet/index.ts#L72

Reply:
> This should really be [...]

Yeah, it seems so :thinking: 

@ST-DDT do you want to automate this? Or is something hard on this?
I would give you an approval for this PR, but it would get removed when you update the PR.
So let me know for what you have decided.